### PR TITLE
Copter/Blimp: Get tune knob channel from RCx_OPTION

### PR DESCRIPTION
**Pull request by muramura**
#20303
I changed the tune knob channel from RCx_OPTION.

### CODE REVIEW

**Review by amilcarlucas — APPROVED** [review]
looks good

File: ArduCopter/radio.cpp [hunk 1]
```diff
@@ -15,7 +15,12 @@ void Copter::default_dead_zones()
     channel_throttle->set_default_dead_zone(30);
     channel_yaw->set_default_dead_zone(20);
 #endif
-    rc().channel(CH_6)->set_default_dead_zone(0);
+    RC_Channel *rc_knobtune = rc().find_channel_for_option(RC_Channel::AUX_FUNC::KNOB_TUNE);
+    if (rc_knobtune == nullptr) {
+        rc().channel(CH_6)->set_default_dead_zone(0);
+    } else {
+        rc_knobtune->set_default_dead_zone(0);
+    }
```
Comment (IamPete1) [inline ArduCopter/radio.cpp:23]: Doesn't look like setting the deadzone does anything as we use the `get_radio_in()` method.

**Review by IamPete1 — CHANGES_REQUESTED** [review]
Need to remove this check:
https://github.com/ArduPilot/ardupilot/blob/069b81819d434a0111df223856b6f78fc0dbeb73/ArduCopter/tuning.cpp#L24-L27

File: ArduCopter/tuning.cpp [hunk 2]
```diff
@@ -9,7 +9,10 @@
 //  should be called at 3.3hz
 void Copter::tuning()
 {
-    const RC_Channel *rc6 = rc().channel(CH_6);
+    RC_Channel *rc6 = rc().find_channel_for_option(RC_Channel::AUX_FUNC::KNOB_TUNE);
+    if (rc6 == nullptr) {
+        rc6 = rc().channel(CH_6);
```
Comment (peterbarker) [inline ArduCopter/tuning.cpp:14]: Let's set the default on RC6_OPTION to be KNOB_TUNE instead of doing this

**Review by peterbarker — CHANGES_REQUESTED** [review]
@muramura I've wanted this for a long time, it would be nice to see it progress.

Even better would be adding support for `KNOB_TUNE2` so people can tune multiple things at once - but that's into the future...

**Comment by peterbarker** [comment]
Sadly this one was neglected.

We've just merged a PR to master which adds this behaviour.

Thanks anyway!

**Pull request closed without merging**